extensions-core/main/runtime/src/main/java/org/apache/camel/quarkus/main/CamelMainRoutesCollector.java

```diff
@@ -66,7 +67,7 @@ public XMLRoutesDefinitionLoader getXmlRoutesLoader() {
         List<RoutesDefinition> answer = new ArrayList<>();
         PackageScanResourceResolver resolver = camelContext.adapt(ExtendedCamelContext.class).getPackageScanResourceResolver();
 
-        for (String part : directory.split(",")) {
+        for (String part : directory.split(",", -1)) {
```

Comment:
Why do we need the empty trailing string here?

Reply:
thx to spot this, I don't really know what's the reason of such change

Reply:
done